src/main/java/ch/uzh/ifi/hase/soprafs21/rest/dto/UserDto.java

```diff
@@ -51,17 +49,10 @@
   @JsonProperty("profilePicture")
   private String profilePicture;
 
-  @JsonProperty("latestLocation")
-  private CoordinateDto latestLocation;
-
```

Comment:
Ditto — removed intentionally? If it's b/c of the rename to lastUserLocation, we should add the new one to the DTO IMO.

Reply:
Generated code. It was removed from API definition that's why it is removed from here.

Reply:
See above issue re. removing this from User/UserDTO

Reply:
the same reply as the previous